content/docs/faq-styling.md

```diff
@@ -28,24 +28,24 @@ render() {
 }
 ```
 
->Tip
+>İpucu
 >
->If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
+>Kendinizi sık sık böyle bir kod yazarken buluyorsanız, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) paketi bunu basitleştirebilir.
 
-### Can I use inline styles? {#can-i-use-inline-styles}
+### Satır içi (inline) stilleri kullanabilir miyim? {#can-i-use-inline-styles}
 
-Yes, see the docs on styling [here](/docs/dom-elements.html#style).
+Evet, stillendirme ile ilgili dokümana [buradan](/docs/dom-elements.html#style) bakın.
 
-### Are inline styles bad? {#are-inline-styles-bad}
+### Satır içi (inline) stiller kötü mü? {#are-inline-styles-bad}
 
-CSS classes are generally better for performance than inline styles.
+CSS sınıfları, performans açısından genellikle satır içi stillerinden daha iyidir.
 
-### What is CSS-in-JS? {#what-is-css-in-js}
+### JS'de CSS (CSS-in-JS) nedir? {#what-is-css-in-js}
```

Comment:
What about `JS-içinde-CSS` ? If you think this is better, can you please apply it for every occurrence ?

Reply:
Sure, thanks.